CHANGELOG.md

```diff
@@ -96,6 +96,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Kenji KATO](https://github.com/katoken-0215)
   [#9706](https://github.com/CocoaPods/CocoaPods/pull/9706)
 
+* Fix XCFramework slice selection when having more slices than requested  
```

Comment:
nit: I think you mean `archs` here, `slice` refers to each individual `.framework` that exists in the xcframework bunde